extensions/integration-tests/setEnvironmentVariables.js

```diff
@@ -81,74 +80,58 @@ if (!fs.existsSync(NOTEBOOK_PYTHON_INSTALL_PATH)) {
-
-// Mapping between AKV secret and the environment variable names
-const SecretEnVarMapping = [];
-SecretEnVarMapping.push([SECRET_AZURE_SERVER, ENVAR_AZURE_SERVER]);
-SecretEnVarMapping.push([SECRET_AZURE_SERVER_PASSWORD, ENVAR_AZURE_SERVER_PASSWORD]);
-SecretEnVarMapping.push([SECRET_AZURE_SERVER_USERNAME, ENVAR_AZURE_SERVER_USERNAME]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER, ENVAR_STANDALONE_SERVER]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER_PASSWORD, ENVAR_STANDALONE_SERVER_PASSWORD]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER_USERNAME, ENVAR_STANDALONE_SERVER_USERNAME]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER_2019, ENVAR_STANDALONE_SERVER_2019]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER_USERNAME_2019, ENVAR_STANDALONE_SERVER_USERNAME_2019]);
-SecretEnVarMapping.push([SECRET_STANDALONE_SERVER_PASSWORD_2019, ENVAR_STANDALONE_SERVER_PASSWORD_2019]);
+const ENVAR_SQL_2017_PASS = 'SQL_2017_PASS';
+const ENVAR_SQL_2019_PASS = 'SQL_2019_PASS';
+const ENVAR_AZURE_SQL_PASS = 'AZURE_SQL_PASS';
 
 // Set the values that are not stored in AKV here
 process.env[ENVAR_PYTHON_INSTALL_PATH] = NOTEBOOK_PYTHON_INSTALL_PATH;
 process.env[ENVAR_RUN_PYTHON3_TEST] = '1';
+process.env[ENVAR_SQL_2017_PASS] = generatePassword();
+process.env[ENVAR_SQL_2019_PASS] = generatePassword();
+process.env[ENVAR_AZURE_SQL_PASS] = generatePassword();
 
-const credential = new DefaultAzureCredential();
-const client = new SecretClient(AKV_URL, credential);
-
-async function main() {
-	for (const entry of SecretEnVarMapping) {
-		const secretName = entry[0];
-		const environmentVariable = entry[1];
-		try {
-			const result = await client.getSecret(secretName);
-			console.log(`${secretName}: ${result.value}`);
-			process.env[environmentVariable] = result.value;
-		} catch (err) {
-			console.error('An error occured while retrieving the value for secret:' + secretName);
-			console.error(err);
-		}
-	}
-	console.log('Done reading values from Azure KeyVault!');
-	console.log(`Launching new window: ${LAUNCH_OPTION}...`);
-	if (LAUNCH_OPTION === LAUNCH_VSCODE) {
-		console.warn('Trying to launch vscode, make sure you have it set properly in the PATH environment variable');
-	}
-	child_process.execSync(LAUNCH_OPTION);
-	console.log('New window for running test has been opened.');
+console.log(`Launching new window: ${LAUNCH_OPTION}...`);
+if (LAUNCH_OPTION === LAUNCH_VSCODE) {
+	console.warn('Trying to launch vscode, make sure you have it set properly in the PATH environment variable');
 }
+execSync(LAUNCH_OPTION);
+console.log('New window for running test has been opened.');
+
+// Start docker containers
+const winargs = ['powershell.exe', '-Command', `Start-Process powershell.exe -ArgumentList '-Command', '$sql2017pass=\\"${process.env[ENVAR_SQL_2017_PASS]}\\";$sql2019pass=\\"${process.env[ENVAR_SQL_2019_PASS]}\\";$azuresqlpass=\\"${process.env[ENVAR_AZURE_SQL_PASS]}\\";${__dirname}\\dockerWindows.ps1' -Verb RunAs`];
```

Comment:
Can you add a comment here describing why you needed to do it this way? (the same sorta thing you replied with to my PR comment)